store/sqlstore/user_store.go

```diff
@@ -1055,20 +1045,41 @@ func (us SqlUserStore) PermanentDelete(userId string) store.StoreChannel {
 	})
 }
 
-func (us SqlUserStore) AnalyticsUniqueUserCount(teamId string) store.StoreChannel {
+func (us SqlUserStore) Count(options model.UserCountOptions) store.StoreChannel {
 	return store.Do(func(result *store.StoreResult) {
-		query := ""
-		if len(teamId) > 0 {
-			query = "SELECT COUNT(DISTINCT Users.Email) From Users, TeamMembers WHERE TeamMembers.TeamId = :TeamId AND Users.Id = TeamMembers.UserId AND TeamMembers.DeleteAt = 0 AND Users.DeleteAt = 0"
+		query := sq.Select("COUNT(Users.Id)").From("Users")
+
+		if !options.IncludeDeleted {
+			query = query.Where("Users.DeleteAt = 0")
+		}
+
+		if options.IncludeBotAccounts {
+			if options.ExcludeRegularUsers {
```

Comment:
nit: While I appreciate that the default case is probably for `IncludeBotAccounts` to be `false` and `ExcludeRegularUsers` to be `false`, having the double negatives seems to make it harder to read. Would it make more sense to converge on `Include*` and just be explicit everywhere we use the `model.UserCountOptions`?